unstable/propertyvalue/propertyvalue.go

```diff
@@ -128,6 +128,22 @@ func RemoveSecrets(pv resource.PropertyValue) resource.PropertyValue {
 	return Transform(unsecret, pv)
 }
 
+func RemoveSecretsAndOutputs(pv resource.PropertyValue) resource.PropertyValue {
```

Comment:
Why are we removing secrets and outputs?

Reply:
The detailed diff algorithm does not care about secrets and outputs, so this allows us to not have to handle them in the code.